proto/options.proto

```diff
@@ -508,8 +512,10 @@ message Options {
   }
   bool linear_references = 45;                                     // Include linear references for graph edges returned in certain responses.
   repeated Costing recostings = 46;                                // CostingType options to use to recost a path after it has been found
-  repeated Ring exclude_polygons = 47;                                    // Rings/polygons to exclude entire areas during path finding
-  bool prioritize_bidirectional = 48;                              // Prioritize bidirectional a*/matrix when depart_at date_time.type is specified [default = false]
+  repeated Ring exclude_polygons = 47;                             // Rings/polygons to exclude entire areas during path finding
+  oneof has_prioritize_bidirectional {
+    bool prioritize_bidirectional = 48;                            // Prioritize bidirectional a*/matrix when depart_at date_time.type is specified [default = false]
+  }
```

Comment:
if it defaults to false we dont need a `oneof`, also seems like its maybe from a different stream of work?